src/vstart.sh

```diff
@@ -1215,9 +1243,9 @@ start_osd() {
     fi
     local osds_wait
     # If the type of OSD is Crimson and the option to balance the Seastar reactors is true
-	if [ "$ceph_osd" == "crimson-osd" ] && [ ! -z "$crimson_balance_cpu" ]; then
+	if [ "$ceph_osd" == "crimson-osd" ]; then
         debug echo "Preparing balance CPU for Crimson"
-        prep_balance_cpu $crimson_smp $crimson_balance_cpu
+        prep_balance_cpu
```

Comment:
We should move the `-z $crimson_balance_cpu ]` check (from within `prep_balance_cpu`) into here.
Otherwise this should be named `maybe_prep_balance_cpu`

Reply:
Moreover, if balancing is **not**  set (dsiabled) - I think that we should fallback to `crimson_seastar_num_threads` (different from `crimson_alien_op_num_threads`).

Meaning, if no specific balancing strategy is used, which is expected for dev work/ simple debug clusters - 
Let's use `crimson_seastar_num_threads` to use the non-pinning strategy.

This is actually _required_ by our bootstrap, if `crimson_seastar_cpu_cores` is not used - `crimson_seastar_num_threads` must be set - otherwise the osd won't come up.

Reply:
discussed offline